unittests/Configuration/Configuration.cpp

```diff
@@ -0,0 +1,40 @@
+#include "vara/Configuration/Configuration.h"
+
+#include "gtest/gtest.h"
+
+namespace vara::feature {
+TEST(Configuration, basicAccessors) {
+  std::string ConfigurationString = R"({"baz":"1","foo":"true"})";
+  auto Config =
+      Configuration::createConfigurationFromString(ConfigurationString);
+  EXPECT_TRUE(Config);
+  std::string Dump = Config->dumpToString();
```

Comment:
I think we should split this test into two, as it tests parsing and dumping.
Test1 should check if we can correctly parse config files.
Where another one, should test if we can correctly dump existing configs.

Maybe also have 1-2 different tests for different scenarios (should there be a bit more complicated ones).